dowhy/causal_prediction/datasets/base_dataset.py

```diff
@@ -1,16 +1,15 @@
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
 
+
 class MultipleDomainDataset:
-    N_STEPS = 5001           # Default, subclasses may override
-    CHECKPOINT_FREQ = 100    # Default, subclasses may override
-    N_WORKERS = 8            # Default, subclasses may override
-    ENVIRONMENTS = None      # Subclasses should override
-    INPUT_SHAPE = None       # Subclasses should override
+    N_STEPS = 5001  # Default, subclasses may override
```

Comment:
strange, why not 5000?

Reply:
This class has been borrowed straight from DomainBed (https://github.com/facebookresearch/DomainBed/blob/main/domainbed/datasets.py#L47) so didn't change numbers. They probably set 5001 so that the loss, acc etc at 5000th step could be printed. Should I change this?